Automate Facebook bot/script.py

```diff
@@ -0,0 +1,51 @@
+import pyautogui
+import time
+import webbrowser
+from selenium import webdriver
+from time import sleep
+from webdriver_manager.chrome import ChromeDriverManager
+from getpass import getpass
+
+usr=input('Enter Email Id:') 
+pwd= getpass('Enter Password:') 
+num = str (input ("Enter comma separated integers: "))
```

Comment:
Comma separated integers?

Reply:
@kaustubhgupta I will change it along with the changes for submit button.

Reply:
@Amit366 okay

Reply:
Umm, why this is reversed? I think you fixed it earlier